Automattic/wp-calypso
# Reader: in search, show 'no sites found' message

#### Changes proposed in this Pull Request

* Show a 'no sites found' message when there are no sites found in a Reader search.

<img width="391" alt="Screen Shot 2021-06-25 at 18 11 47" src="https://user-images.githubusercontent.com/17325/123378811-5546e700-d5e1-11eb-8395-447dc3534590.png">

Props to @Saravanan90 for his earlier work on this in https://github.com/Automattic/wp-calypso/pull/45144.

#### Testing instructions

* Test Reader search at a variety of breakpoints and on various browsers: http://calypso.localhost:3000/read/search.
* Ensure that, when there are no site results, the "No sites found." message is displayed.

Related to https://github.com/Automattic/wp-calypso/issues/16603.
